staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -94,27 +97,26 @@ func getResourceHandler(scope RequestScope, getter getterFunc) http.HandlerFunc
 		ctx := scope.ContextFunc(req)
 		ctx = request.WithNamespace(ctx, namespace)
 
-		result, err := getter(ctx, name, req)
+		result, err := getter(ctx, name, req, trace)
 		if err != nil {
 			scope.err(err, w, req)
 			return
 		}
+		trace.Step("About to create self-link")
```

Comment:
Maybe - but yesterday on slack there was a discussion about some Gets taking over a second. Without traces it's pretty hard to tell where the problem was.